### DISCUSSION TITLE
Deployments stuck in Queued

### DISCUSSION THREAD

**Discussion by hieuhani** [vercel/community#3352] [Help] [answered]
### Summary

I was trying to deploy my project that is imported from Github and it's been stucked at Queued for 19h. Does my account reach the limitation or where can I see the limitation or when it's possible to continue to run?

I've checked to vercel-status, it's all operational

[nuxt-multi-tenancy-7vv6c2fz2-hieuhani.vercel.app](https://vercel.live/link/nuxt-multi-tenancy-7vv6c2fz2-hieuhani.vercel.app?via=deployment-domains-list-commit)

### Example

https://vercel.com/hieuhani/nuxt-multi-tenancy/J7EFn2aQZMHxcuAmFAAVSsS6zLWt

### Steps to Reproduce

I was trying to create a new project that is imported from Github, then configure the build command as normal

**Comment by amyegan** [comment]
Hey @hieuhani. Does it work if you use the CLI to deploy from a local build? The steps to do that can be found at https://vercel.com/docs/cli/deploying-from-cli#deploying-from-local-build-prebuilt

**Reply by hieuhani** [reply]
Thank for responding.

It's still on queued in my local build for 5 minutes https://vercel.com/hieuhani/nuxt-multi-tenancy/6g8z6ZAu7djKaPY4z1fnabmUkrPJ

```
nuxt-multi-tenancy git:(main) vercel build
✨  Done in 10.19s.
✅  Build Completed in .vercel/output [11s]


nuxt-multi-tenancy git:(main) ✗ vercel deploy --prebuilt
Vercel CLI 31.0.4
🔍  Inspect: https://vercel.com/hieuhani/nuxt-multi-tenancy/AfnLtfY45NnwvpdgJDxSEkngHBQ1 [1s]
⠏ Queued
```

**Comment by swarnava** [comment, marked as answer]
We had some issue with Hobby user which we fixed internally. Can you push a new commit or deploy using CLI and let us know if you still seeing this behavior?

**Reply by hieuhani** [reply]
Thank you very much! It works now!

**Comment by toniuyt123** [comment]
I'm having the same issue. Example: https://vercel.com/toniuyt123/dark-meta/D3Cihs4aCAjE2igiAgLrNnuP1Upo
Any ideas? Vercel-status says it's all operational

**Reply by swarnava** [reply]
Looks like it has deployed now

**Reply by HenryHaber** [reply]
Having similar issues with my deployment

**Comment by AnuragRamdasan** [comment]
Hi this issue continues to be the case for me. Tried a manual deploy but that's stuck in queue too. Is there a way to resolve this?

**Comment by swarnava** [comment]
Please share the deployment id if you are still facing issue

**Reply by umair-vettel** [reply]
Facing same issue. Here's the deploymnet ID b2byizp14

**Reply by swarnava** [reply]
That looks like a commit ID, please go to Vercel Dashboard > Teams > Project > Deployment and share us the unique id or url

**Reply by jhigger** [reply]
```rh8l1ydob```

**Comment by simonMoisselin** [comment]
same here for me,
here is the deployment id `chat-resume-ck3wkt77p`

**Comment by erwin-io** [comment]
Same issue here is the deployment id m6ee1ayah

**Comment by jhigger** [comment]
same issue ```cod9bojko```

**Reply by jhigger** [reply]
fixed

**Comment by gisbelt** [comment]
Same here: rdytka6hy

**Reply by gisbelt** [reply]
It works, deployed now.

**Comment by nikhiljay** [comment]
Same here (on the hobby plan). There's clearly a problem right now even though "All Systems Operational"

**Reply by gisbelt** [reply]
Yes, it seems like an internal issue
